webviz_config/webviz_store.py

```diff
@@ -25,7 +26,7 @@ def register_function(self, func):
 
         return_type = inspect.getfullargspec(func).annotations["return"]
 
-        if return_type not in [pd.DataFrame, pathlib.Path]:
+        if return_type not in [pd.DataFrame, pathlib.Path, io.BytesIO]:
             raise NotImplementedError(
                 "Currently only storage of dataframes "
                 "and file resources are implemented."
```

Comment:
Update error message. Maybe we can use this opportunity to move the expanding list `[pd.DataFrame, pathlib.Path, io.BytesIO]` to a constant, and then use that constant both in the `if` statement, and in the `NotImplementedError`.